pytorch_lightning/loggers/neptune.py

```diff
@@ -199,12 +230,23 @@ def __init__(
             name: Optional[str] = None,
             run: Optional[str] = None,
             prefix: str = '',
+            base_namespace: str = None,
             **neptune_run_kwargs):
         if neptune is None:
             raise ImportError(
                 'You want to use `neptune` in version >=0.9 logger which is not installed yet,'
                 ' install it with `pip install "neptune-client>=0.9"`.'
             )
+        used_legacy_kwargs = [
+            legacy_kwarg for legacy_kwarg in neptune_run_kwargs.keys()
+            if legacy_kwarg in LEGACY_NEPTUNE_INIT_KWARGS
+        ]
+        if used_legacy_kwargs:
+            raise ValueError(
+                f"Following kwargs used by you are deprecated: {used_legacy_kwargs}."
+                f" You should use arguments accepted by either `NeptuneLogger.init` or `neptune.init`."
```

Comment:
@shnela let's change this to:
Following kwargs used by you are deprecated: {used_legacy_kwargs}.
If you are looking for the Neptune logger using legacy Python API it has been renamed to NeptuneLegacyLogger. The NeptuneLogger was re-written to use the neptune.new Python API (learn more: https://neptune.ai/blog/neptune-new).
You should use arguments accepted by either `NeptuneLogger.init` or `neptune.init`

Reply:
ok